lib/pure/selectors.nim

```diff
@@ -312,6 +312,8 @@ else:
     include ioselects/ioselectors_poll # need to replace it with event ports
   elif defined(genode):
     include ioselects/ioselectors_select # TODO: use the native VFS layer
+  elif defined(nintendoswitch):
+    include ioselects/ioselectors_select
```

Comment:
doesn't the os have something better than select? kqueue, epoll?

Reply:
Unfortunately, not that I can find. If you can find it in the devkitPro libs, I would be up for changing it.